app/helpers/translatable_form_helper.rb

```diff
@@ -47,7 +51,12 @@ def translatable_field(field_type, method, options = {})
               final_options = @template.merge_translatable_field_options(options, locale)
                                        .reverse_merge(label: label_without_locale)
 
-              @template.concat send(field_type, localized_attr_name, final_options)
+              if field_type == :cktext_area
+                @template.concat content_tag :span, send(field_type, localized_attr_name, final_options),
+                                             @template.merge_translatable_field_options(options, locale)
```

Comment:
I thought it would not be a problem the label_options attributes, but if you don't mind duplication, that's fine! https://github.com/consul/consul/pull/2914/commits/4344165bce7b5009e1635da47295cbbc3631d37e